timeoff/urls.py

```diff
@@ -1,7 +1,10 @@
 from django.urls import path
 from rest_framework import routers
 
-from .views import list_all_timeoff_requests, list_user_timeoff_requests, submit_timeoff_view
+from .views import list_all_timeoff_requests
```

Comment:
Why are the imports from the same place imported over multiple lines?

Reply:
Will change this.